collector/kernel/buffered_poller.cc

```diff
@@ -79,7 +80,7 @@ BufferedPoller::BufferedPoller(
       nat_handler_(writer_, log),
       nic_poller_(nic_poller),
       tcp_socket_table_ever_full_(false),
-      tslot_(1e9, 16),
+      tslot_(socket_stats_interval_sec * 1000 * 1000 * 1000, 16),
```

Comment:
Any chance we can convert 1k*1k*1k to a #define constant?

Reply:
Is tslot_ actually expecting a double, or an unsigned long - and we're just doing a silent conversion?

Reply:
Yes, I could add #defines but there are multiple places in the code that use 1000 * 1000, 1000 * 1000 * 1000 or similar, probably don't want to change them all with this changeset.  Maybe makes sense as a separate changeset.

tslot_ is a fast_div which has a ctor that takes a double: inline fast_div::fast_div(double amt, uint32_t required_bits)